samples/bookinfo/networking/egress-rule-google-apis.yaml

```diff
@@ -4,18 +4,43 @@ metadata:
   name: googleapis
 spec:
   hosts:
-  - "*.googleapis.com"
+  - www.googleapis.com
   ports:
+  - number: 80
+    name: http
+    protocol: HTTP
   - number: 443
     name: https
-    protocol: http
+    protocol: HTTPS
+  resolution: DNS
+---
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: rewrite-port-for-googleapis
+spec:
+  hosts:
+  - www.googleapis.com
+  http:
+  - match:
+    - port: 80
+    route:
+    - destination:
+        host: www.googleapis.com
+        port:
+          number: 443
```

Comment:
Also, I don't understand why this VS is required

Reply:
Because he's configuring tls origination